modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/RESTAPITestConstants.java

```diff
@@ -121,7 +121,7 @@
     public static final String BASIC_AUTH_HEADER = "admin:admin";
 
     //client-registration link
-    public static final String CLIENT_REGISTRATION_URL = "client-registration/v0.13/register";
+    public static final String CLIENT_REGISTRATION_URL = "client-registration/v0.14/register";
```

Comment:
Can we reuse the REST_API_VERSION constant here so that we dont have to modify these tests?